nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -552,3 +552,5 @@ The module update takes care of the new config syntax and the data itself (user
 - `teleport` has been upgraded from major version 12 to major version 14. Please see upstream [upgrade instructions](https://goteleport.com/docs/management/operations/upgrading/) and release notes for versions [13](https://goteleport.com/docs/changelog/#1300-050823) and [14](https://goteleport.com/docs/changelog/#1400-092023). Note that Teleport does not officially support upgrades across more than one major version at a time. If you're running Teleport server components, it is recommended to first upgrade to an intermediate 13.x version by setting `services.teleport.package = pkgs.teleport_13`. Afterwards, this option can be removed to upgrade to the default version (14).
 
 - The Linux kernel module `msr` (see [`msr(4)`](https://man7.org/linux/man-pages/man4/msr.4.html)), which provides an interface to read and write the model-specific registers (MSRs) of an x86 CPU, can now be configured via `hardware.cpu.x86.msr`.
+
+- `trilium-desktop` and `trilium-server` have been updated to [v0.61](https://github.com/zadam/trilium/releases/tag/v0.61.11). For existing installations, upgrading to this version is supported only from v0.60.X. If an older version is used, make sure to update to v0.60 (available in NixOS 22.05) first and only then to v0.61.
```

Comment:
```suggestion
- `trilium-desktop` and `trilium-server` have been updated to [v0.61](https://github.com/zadam/trilium/releases/tag/v0.61.11). For existing installations, upgrading to this version is supported only after running v0.60.X at least once. If you are still on an older version, make sure to update to v0.60 (available in NixOS 23.05) first and only then to v0.61 (available in NixOS 23.11).
```
It's 2023 :p

Reply:
Ha, must have been tired.

I've updated the release note and updated to 0.61.12.

Reply:
0.61.13 is out: https://github.com/zadam/trilium/releases/tag/v0.61.13 with a fix for the

> sync error with revisions (the underlying cause is unknown, but a more graceful error handling should mitigate the problem)

Reply:
@FliegendeWurst , done.